spec/lib/rbac/query_shared_resource_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe RBAC::QuerySharedResource do
+  let(:app_name) { 'catalog' }
+  let(:resource) { "portfolios" }
+  let(:verbs) { ["read"] }
+  let(:resource_id1) { "1" }
```

Comment:
Similar to my comment about `role1_uuid` and `role2_uuid`, I think this could just be hardcoded down on line 28.